### PR TITLE
Update issue templates

### LINKED ISSUE #85: Add issue templates

### PR DESCRIPTION
**Pull request by ShishirH**
Closes #85

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (dinogun) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]: This appears to be specific to Grafana ? Can you please make it more generic and maybe add reference to the APIs being used.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (dinogun) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: Instead of Desktop, it should be "Environment"

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (ShishirH) [reply .github/ISSUE_TEMPLATE/bug_report.md:13]: @dinogun Should I add some generic steps, or should I leave it at Steps to reproduce the behavior?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (ShishirH) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Changed Desktop to Environment

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (dinogun) [reply .github/ISSUE_TEMPLATE/bug_report.md:13]: `Steps to reproduce` should be good enough

Comment (ShishirH) [reply .github/ISSUE_TEMPLATE/bug_report.md:13]: Made the change

**Pull request merged**